handler/graphql.go

```diff
@@ -279,6 +281,102 @@ func GraphQL(exec graphql.ExecutableSchema, options ...Option) http.HandlerFunc
 	return handler.ServeHTTP
 }
 
+func TyphonHandlerFromGraphQL(exec graphql.ExecutableSchema, options ...Option) typhon.Service {
```

Comment:
Rather than support all transports forever we should work how to make handler open for extension.

The way this is setup currently, everyone using gqlgen would need to fetch `monzo/typoon`.